frontend/public/components/utils/copy-to-clipboard.jsx

```diff
@@ -1,15 +1,65 @@
 import * as React from 'react';
 
 import { CopyToClipboard as CTC } from 'react-copy-to-clipboard';
+import { Tooltip } from 'react-lightweight-tooltip';
 
-export const CopyToClipboard = ({value, visibleValue = value}) => <React.Fragment>
-  <div className="co-copy-to-clipboard">
-    <pre className="co-pre-wrap co-copy-to-clipboard__text">{visibleValue}</pre>
-    <CTC text={value}>
-      <button className="btn btn-default co-copy-to-clipboard__btn" type="button">
-        <i className="fa fa-clipboard" aria-hidden="true"></i>
-        <span className="sr-only">Copy to Clipboard</span>
-      </button>
-    </CTC>
-  </div>
-</React.Fragment>;
+export class CopyToClipboard extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {copied: false};
+  }
+
+  render() {
+    const overrides = {
+      wrapper: {
+        position: 'absolute',
+        right: '0px',
+        top: '0px',
+        width: '42px',
+        height: '42px',
+      },
+      tooltip: {
+        left: '-90px',
+        bottom: '-6px',
+        maxWidth: '150px',
+        minWidth: '150px',
+        padding: '0',
+        textAlign: 'center',
+      },
+      content: {
+        display: 'block',
+        fontSize: '12px',
```

Comment:
add `fontFamily: '"Open Sans",Helvetica,Arial,sans-serif',` as the current font is the monospace stack that's being inherited from `<pre>`.

Reply:
If the styles really are specific to this component, I'd be OK keeping them in this file just for the copy button instead of trying to generalize them.